From 4d2e39342352fce3969064e58e366753e7002f46 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Tue, 15 Feb 2022 01:21:01 +0000
Subject: [PATCH] packit: build on and use Fedora 35 spec file

It's targeted to the v249 branch, while the rawhide one follows
the newest upstream release, and the command line options are not
compatible

Conflict:NA
Reference:https://github.com/systemd/systemd/commit/4d2e39342352fce3969064e58e366753e7002f46
---
 .packit.yml | 12 +++++-------
 1 file changed, 5 insertions(+), 7 deletions(-)

diff --git a/.packit.yml b/.packit.yml
index ee47f8954e..fb2cb5a7c3 100644
--- a/.packit.yml
+++ b/.packit.yml
@@ -16,8 +16,8 @@ upstream_tag_template: "v{version}"
 
 actions:
   post-upstream-clone:
-    # Use the Fedora Rawhide specfile
-    - "git clone https://src.fedoraproject.org/rpms/systemd .packit_rpm --depth=1"
+    # Use the Fedora 35 specfile
+    - "git clone --branch f35 https://src.fedoraproject.org/rpms/systemd .packit_rpm --depth=1"
     # Drop the "sources" file so rebase-helper doesn't think we're a dist-git
     - "rm -fv .packit_rpm/sources"
     # Drop backported patches from the specfile, but keep the downstream-only ones
@@ -31,14 +31,12 @@ actions:
     # [0] https://github.com/mesonbuild/meson/issues/7360
     # [1] https://github.com/systemd/systemd/pull/18908#issuecomment-792250110
     - 'sed -i "/^CONFIGURE_OPTS=(/a--werror" .packit_rpm/systemd.spec'
-    # cryptolib is supported from v250 and newer, remove it
-    - 'sed -i "/-Dcryptolib=openssl/d" .packit_rpm/systemd.spec'
 
 jobs:
 - job: copr_build
   trigger: pull_request
   metadata:
     targets:
-    - fedora-rawhide-aarch64
-    - fedora-rawhide-i386
-    - fedora-rawhide-x86_64
+    - fedora-35-aarch64
+    - fedora-35-i386
+    - fedora-35-x86_64
-- 
2.33.0

